app/data/census_individual.json

```diff
@@ -160,11 +79,11 @@
                 },
                 {
                     "id": "date-of-birth",
-                    "title": "John Doe",
+                    "title": "{{ answers.person_full_name }}",
```

Comment:
we don't appear to use it in the html title. however it is mandatory in the schema. should this be changed?